core/forage-core-common/pom.xml

```diff
@@ -26,6 +26,10 @@
             <groupId>dev.langchain4j</groupId>
             <artifactId>langchain4j-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-jbang-core</artifactId>
```

Comment:
We shouldn't bundle this on forage-common, because then it will be part of the depencies for all runtimes. I think we need to move jbang-related code to a separate module that is included only when needed.

Reply:
fixed